Monika After Story/game/script-topics.rpy

```diff
@@ -9393,3 +9393,116 @@ label monika_cares_about_dokis:
             m 1hub "Ahahaha!"
 
     return "derandom"
+
+default persistent._mas_pm_live_north_hemisphere = None
```

Comment:
Looking at the season functions, it would be better if this persistent tracks southern hemisphere instead of north. 

That way we could just check `if persistent._mas_pm_live_south_hemisphere:` instead of checking if the reverse is not False.